<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>613</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1280</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>颜色设置</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Red">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Red">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Green">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Green">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Blue">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Blue">
         <property name="maximum">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_Alpha">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="Alpha">
         <property name="maximum">
          <number>255</number>
         </property>
         <property name="value">
          <number>255</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="label">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>画笔大小</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="width">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>选区</string>
    </property>
    <addaction name="actionSelect"/>
    <addaction name="actionDeleteSelect"/>
    <addaction name="actionMove"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>绘图</string>
    </property>
    <addaction name="actionPen"/>
    <addaction name="actionEraser"/>
    <addaction name="actionRect"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>效果</string>
    </property>
    <addaction name="actionBlur"/>
    <addaction name="actionSharpen"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionPen"/>
   <addaction name="actionEraser"/>
   <addaction name="actionRect"/>
   <addaction name="separator"/>
   <addaction name="actionSharpen"/>
   <addaction name="actionBlur"/>
   <addaction name="separator"/>
   <addaction name="actionSelect"/>
   <addaction name="actionDeleteSelect"/>
   <addaction name="actionMove"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/打开.png</normaloff>:/icon/打开.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/保存.png</normaloff>:/icon/保存.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/虚线.png</normaloff>:/icon/虚线.png</iconset>
   </property>
   <property name="text">
    <string>矩形选区</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="text">
    <string>椭圆</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/画笔.png</normaloff>:/icon/画笔.png</iconset>
   </property>
   <property name="text">
    <string>画笔</string>
   </property>
   <property name="toolTip">
    <string>P</string>
   </property>
   <property name="shortcut">
    <string>P</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/橡皮.png</normaloff>:/icon/橡皮.png</iconset>
   </property>
   <property name="text">
    <string>橡皮</string>
   </property>
   <property name="toolTip">
    <string>E</string>
   </property>
   <property name="shortcut">
    <string>E</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="text">
    <string>模糊</string>
   </property>
  </action>
  <action name="actionSharpen">
   <property name="text">
    <string>锐化</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icon/新建.png</normaloff>:/icon/新建.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
   <property name="toolTip">
    <string>Ctrl+N</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDeleteSelect">
   <property name="text">
    <string>删除选区</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="icon">
    <iconset theme="R" resource="res.qrc">
     <normaloff>:/icon/绘制矩形.png</normaloff>:/icon/绘制矩形.png</iconset>
   </property>
   <property name="text">
    <string>矩形</string>
   </property>
   <property name="shortcut">
    <string>R</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>移动选区</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
